crypto/secp256k1/secp256k1_cgo.go

```diff
@@ -1,23 +0,0 @@
-// +build libsecp256k1
```

Comment:
crypto/secp256k1/secp256k1.go is used as nocgo, right?
In BSC side, nocgo is not used, BSC use secp256k1_nocgo.go, which is removed in this PR, is it reasonable?